server/swap/swap.go

```diff
@@ -28,6 +28,11 @@ var (
 	// txWaitExpiration is the longest the Swapper will wait for a coin waiter.
 	// This could be thought of as the maximum allowable backend latency.
 	txWaitExpiration = time.Minute
+
+	requestExpiration = 30 * time.Second
```

Comment:
Since request expiration is not set in the call to `AuthManager.Request`, the `Swapper` gets to select an appropriate request expiration time.